nutils/expression_v2.py

```diff
@@ -709,6 +709,26 @@ def define_for(self, __name: str, *, gradient: Optional[str] = None, curl: Optio
                 raise ValueError('Cannot define the jacobian {!r}: dimension is negative.'.format(jacobian))
             setattr(self, jacobian, function.jacobian(geom, numpy.size(geom) - i))
 
+    def add_field(self, __names: tuple, __bases: tuple = (), shape: Tuple[int, ...] = (), dtype: function.DType = float):
```

Comment:
The annotation for `names` should be `Union[str, Sequence[str]]`. The annotation for `bases` is incorrect as well. Perhaps it's best to remove it.

Reply:
fixed (and turned bases into varargs)